.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,4 @@
+name: Bug Report
+description: File a bug report
```

Comment:
Are you sure this template is valid? According to the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax):

> All issue form configuration files must begin with `name`, `description`, and `body` key-value pairs.

Reply:
Good catch! I'm leaving it as an empty array

Reply:
I opened another PR here with templates auto-generated by Github: https://github.com/cognitedata/cognite-sdk-js/pull/945